package.json

```diff
@@ -36,9 +36,8 @@
     "pretest": "npm run build",
     "posttest": "agadoo internal/index.mjs",
     "prepublishOnly": "export PUBLISH=true && npm test && npm run create-stubs",
```

Comment:
Need to remove the `npm run create-stubs` call here

Reply:
Ops. Nice catch. 👍